zio-http/shared/src/main/scala/zio/http/Body.scala

```diff
@@ -569,7 +569,7 @@ object Body {
       ZStream.unwrap {
```

Comment:
I don't think we need this right? Since we're already wrapping the for-comprehension in `blocking`

Reply:
I think we do as the for comprehensive returns a `ZIO[_, _, ZStream[...]]` 🤔